wesnoth-sota.po

```diff
@@ -818,6 +819,12 @@ msgid ""
 "defensive, and while his hatred of me was unabated, his desire to live made "
 "him somewhat maleable."
 msgstr ""
+"22. III. roku 23 w. l.:\n"
```

Comment:
v předloze je 23 zápisů bez dvojtečky a 6 s dvojtečkou
dovolil jsem si to v překladu [sladit](fe3fd10404945a97c336b24f4cefac0b3398bf6f) na všechny bez dvojtečky (přijde mi to čitelnější)
```suggestion
"22. III. roku 23 w. l.\n"
```